test/modules/aws-account-integration.ts

```diff
@@ -55,7 +55,7 @@ describe('AwsAccount Integration Testing', () => {
   it('does absolutely nothing when you sync this', sync());
 
   it('does absolutely nothing when you plan this', query(`
-    select iasql_plan();
+    select iasql_plan_apply();
```

Comment:
would be nice to do a quick check and select that no records are return. not blocking though

Reply:
🙌